src/snac/client/webui/WebUIExecutor.php

```diff
@@ -1364,7 +1364,45 @@ public function displayMessageListPage(&$display) {
         if (!isset($serverResponse["result"]) || $serverResponse["result"] != 'success')
             return $this->drawErrorPage($serverResponse, $display);
 
-        $display->setData($serverResponse);
+        $data = $serverResponse + ["viewSetting" => "inbox"];
+        $display->setData($data);
+        $display->setTemplate("message_list");
+    }
+
+    /**
+     * Display Archived Messages
+     *
+     * Asks the server for user's archived messages, then loads the display with the message
+     * center (message list) template and sets the message center viewSetting
+     *
```

Comment:
The PHPDoc needs the @param annotation.  It can be copied from one of the other methods that take display